docs/docs/workflows/verify-cli.md

```diff
@@ -40,6 +40,30 @@ tlog entry verified with uuid: afaba7f6635b3e058888692841848e5514357315be9528474
 Verified OK
 ```
 
+## Verify the provenance
+
+Provenance is a positive attestation that a software artifact was produced by a specific repository and build system invocation. For more information on provenance visit [slsa.dev](https://slsa.dev/provenance/v0.2).
```

Comment:
Is "positive attestation" a known phrase? Otherwise I'd rephrase to something like:
```suggestion
Provenance attests that a software artifact was produced by a specific repository and build system invocation. For more information on provenance visit [slsa.dev](https://slsa.dev/provenance/v0.2).
```